system/jlib/jtrace.cpp

```diff
@@ -1142,7 +1146,7 @@ class CSpan : public CInterfaceOf<ISpan>
 
     opentelemetry::trace::StartSpanOptions opts;
     nostd::shared_ptr<opentelemetry::trace::Span> span;
-
+    std::vector<std::pair<std::string, opentelemetry::common::AttributeValue>> pendingAttributes;
```

Comment:
Should be in the derived class.

Reply:
definitely, this line actually needs to be removed. The derived class includes this and the numeric pending attributes:
https://github.com/rpastrana/HPCC-Platform/blob/0df9dcca53ca4bed084b0371756385260cfd9b19/system/jlib/jtrace.cpp#L1370